paper.tex

```diff
@@ -166,15 +166,18 @@ \section*{Introduction}
 simulators~\citep[e.g.,][]{ewing2010msms,chen2009fast,staab2015scrm}.
 However, this conciseness means that models of even intermediate complexity
 are difficult for humans to understand, making errors likely.
-Input parameter file formats for simulators allow the model specification
+APIs are more verbose, but require a substantial time investment to
+learn, and are tied to a specific tool, so this knowledge is not portable
```

Comment:
I feel like there's one too many commas here, but can't decide which one should go?